### ISSUE TITLE
tibia terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3728]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased bone mineral content of tibia
2. decreased bone mineral content of tibia
3. increased volumetric bone mineral density of the midshaft of tibia
4. decreased volumetric bone mineral density of the midshaft of tibia
5. increased volumetric bone mineral density of proximal tibia
6. decreased volumetric bone mineral density of proximal tibia
7. increased compact volumetric bone mineral density of proximal tibia
8. decreased compact volumetric bone mineral density of proximal tibia
9. increased trabecular volumetric bone mineral density of proximal tibia
10. decreased trabecular volumetric bone mineral density of proximal tibia
11. increased ultimate load of tibia
12. decreased ultimate load of tibia
13. abnormal strength of tibia

>2. Synonyms (e.g., absent spleen)

1. increased tibia bone mineral content
2. decreased tibia bone mineral content
3. increased midshaft tibia volumetric bone mineral density
4. decreased midshaft tibia volumetric bone mineral density
5. increased proximal tibia volumetric bone mineral density
6. decreased proximal tibia volumetric bone mineral density
7. increased proximal tibia cortical volumetric bone mineral density; increased cortical volumetric bone mineral density of proximal tibia
8. decreased proximal tibia cortical volumetric bone mineral density; increased cortical volumetric bone mineral density of proximal tibia
9. increased proximal tibia trabecular volumetric bone mineral density
10. decreased proximal tibia trabecular volumetric bone mineral density
11. increased tibia ultimate load
12. decreased tibia ultimate load
13. abnormal tibia strength

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. elevation in the amount (usually in grams/cm) of bone mineral divided by a bone-scanned area in the tibia
2. reduction in the amount (usually in grams/cm) of bone mineral divided by a bone-scanned area in the tibia
3. elevation in the qualitative measurement value of mineral content of bone in the medial and larger of the two bones of the lower leg, measured in the middle of the diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
4. reduction in the qualitative measurement value of mineral content of bone in the medial and larger of the two bones of the lower leg, measured in the middle of the diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
5. elevation in the qualitative measurement value of mineral content of bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
6. reduction in the qualitative measurement value of mineral content of bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
7. elevation in the qualitative measurement value of mineral content of compact bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
8. reduction in the qualitative measurement value of mineral content of compact bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
9. elevation in the qualitative measurement value of mineral content of trabecular bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
10. reduction in the qualitative measurement value of mineral content of trabecular bone in the medial and larger of the two bones of the lower leg, measured in the proximal diaphysis. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
11. elevation of the maximum force a tibia can tolerate before fracturing under axial compression.
12. reduction of the maximum force a tibia can tolerate before fracturing under axial compression.
13. a change related to the ability of a tibia to withstand the application of force without yielding or breaking.

PMID:16269906; https://phys.libretexts.org/Bookshelves/Conceptual_Physics/Book%3A_Body_Physics_-_Motion_to_Metabolism_(Davis)/07%3A_Strength_and_Elasticity_of_the_Body/7.01%3A_Strength_of_Human_Bones

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. increased bone mineral content (MP:0010123)
2. decreased bone mineral content (MP:0010124)
3. increased volumetric bone mineral density of tibia (new term)
4. decreased volumetric bone mineral density of tibia (new term)
5. increased volumetric bone mineral density of tibia (new term)
6. decreased volumetric bone mineral density of tibia (new term)
7. increased volumetric bone mineral density of tibia (new term)
8. decreased volumetric bone mineral density of tibia (new term)
9. increased volumetric bone mineral density of tibia (new term)
10. decreased volumetric bone mineral density of tibia (new term)
11. abnormal strength of tibia (new term)
12. abnormal strength of tibia (new term)
13. abnormal bone strength (new term in issue #3727)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by slaulederkind** [comment]
Preferred term labels 3. to 6. should be:
3. increased total volumetric bone mineral density of the midshaft of tibia
4. decreased total volumetric bone mineral density of the midshaft of tibia
5. increased total volumetric bone mineral density of proximal tibia
6. decreased total volumetric bone mineral density of proximal tibia

Suggested parent terms should be:

3. increased volumetric bone mineral density (MP:0013615)
4. decreased volumetric bone mineral density (MP:0013616)
5. increased volumetric bone mineral density (MP:0013615)
6. decreased volumetric bone mineral density (MP:0013616)
7. increased total volumetric bone mineral density of proximal tibia (new term)
8. decreased total volumetric bone mineral density of proximal tibia (new term)
9. increased total volumetric bone mineral density of proximal tibia (new term)
10. decreased total volumetric bone mineral density of proximal tibia (new term)

**Comment by sbello** [comment]
@slaulederkind similar question to #3729 how essential to the phenotype is the location of the measurement and how essential is the specific bone? Do you have reports of differences in the phenotype in the same model if they measure in the midshaft or the proximal region? Is the tibia a stand in for long bones in general?
